tests/python/contrib/test_hexagon/test_launcher.py

```diff
@@ -40,6 +42,251 @@
+
+                        rpc_info = {
+                            "rpc_tracker_host": tvm_tracker_host,
+                            "rpc_tracker_port": tvm_tracker_port,
+                            "rpc_server_port": RPC_SERVER_PORT + 0,  # See note at the beginning of the file
+                            "adb_server_socket": adb_server_socket,
+                        }
+                        launcher = HexagonLauncher(serial_number=android_serial_number, rpc_info=rpc_info)
+                        launcher.upload(str(host_dso_binary_path), str(target_dso_binary_filename))
+                        launcher.start_server()
+
+                        try:
+                            with launcher.start_session() as sess:
+                                mod = launcher.load_module(target_dso_binary_filename, sess)
+
+                                host_numpy_A_data = np.ndarray(shape, dtype=dtype)
+                                host_numpy_B_data = np.ndarray(shape, dtype=dtype)
+                                host_numpy_C_data = np.ndarray(shape, dtype=dtype)
+                                host_numpy_C_data_expected = np.ndarray(shape, dtype=dtype)
+
+                                def intended_val_A(i,j):
+                                    return i + j
+
+                                def intended_val_B(i,j):
+                                    return (i+1) * (j+1)
+
+                                for i in range(shape[0]):
+                                    for j in range(shape[1]):
+                                        host_numpy_A_data[i,j] = intended_val_A(i,j)
+                                        host_numpy_B_data[i,j] = intended_val_B(i,j)
+                                        host_numpy_C_data_expected[i,j] = intended_val_A(i,j) + intended_val_B(i,j)
+
+                                A_data = tvm.nd.empty(shape, dtype, sess.device, mem_scope)
+                                A_data.copyfrom(host_numpy_A_data)
+
+                                B_data = tvm.nd.empty(shape, dtype, sess.device, mem_scope)
+                                B_data.copyfrom(host_numpy_B_data)
+
+                                C_data = tvm.nd.empty(shape, dtype, sess.device, mem_scope)
+
+                                timer = mod.time_evaluator("add_hvx", sess.device, number=100, repeat=1, min_repeat_ms=1000)
+                                timing_result = timer(A_data, B_data, C_data)
+
+                                print("TIMING RESULT: {}".format(timing_result))
+
+                                # Verify that the computation actually happened, and produced the correct result.
+                                result = C_data.numpy()
+                                assert (result == host_numpy_C_data_expected).all()
+
+                                br.record_success(dtype, sched_type, mem_scope, num_vectors_per_tensor, timing_result)
+                        except:
```

Comment:
Prefer `except Exception` rather than a bare `except`.  A bare `except` will catch much more, including [`KeyboardInterrupt`](https://docs.python.org/3/library/exceptions.html#KeyboardInterrupt), making it impossible to Ctrl-C out of the entire script.